src/api.rs

```diff
@@ -211,53 +210,42 @@ impl fmt::Display for SentryError {
     }
 }
 
+#[derive(Clone, Debug, thiserror::Error)]
+#[error("project was renamed to '{0}'\n\nPlease use this slug in your .sentryclirc or sentry.properties or for the --project parameter")]
```

Comment:
Why the double newlines `\n\n`?

```suggestion
#[error("project was renamed to '{0}'\n\nPlease use this slug in your .sentryclirc file, sentry.properties file or in the CLI --project parameter")]
```

Reply:
It used to be 2 `\n\n` and I didn't want to change it unnecessarily. But now that I'll use your improved wording, I can bump it too.